src/schemas/json/package.json

```diff
@@ -196,6 +196,42 @@
           "description": "Note that the \"typings\" field is synonymous with \"types\", and could be used as well.",
           "type": "string"
         },
+        "typesVersions": {
+          "description": "The \"typesVersions\" field is used since TypeScript 3.1 to support features that were only made available in newer TypeScript versions.",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
This should be using `patternProperties`, but the syntax for this [is rather complex](https://github.com/npm/node-semver#ranges).